workflow/executor/docker/docker.go

```diff
@@ -249,7 +259,7 @@ func (d *DockerExecutor) syncContainerIDs(ctx context.Context, containerNames []
 			}
 			// sidecars start after the main containers, so we can't just exit once we know about all the main containers,
 			// we need a bit more time
-			if d.haveContainers(containerNames) && time.Since(started) > 3*time.Second {
+			if d.haveContainers(containerNames) && time.Since(started) > 30*time.Second {
```

Comment:
@whynowy @jessesuen I've increased this to up-to 30s (some for PNS). I'm assuming that all sidecar containers will start within 30s. Is this a valid assumption?